arm-network/2016-06-01/swagger/network.json

```diff
@@ -7234,6 +7234,49 @@
       ],
       "description": "IPConfiguration"
     },
+    "ResourceNavigationLinkFormat": {
+      "properties": {
+        "linkedResourceType": {
+          "type": "string",
+          "description": "Gets or sets the resource type of the linked resource"
+        },
+        "link": {
+          "type": "string",
+          "description": "Gets or sets the link to the external resource"
+        },
+        "parentResourceGuid": {
+          "type": "string",
+          "description": "Gets or sets the link to the external resource"
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets or sets Provisioning state of the ResourceNavigationLink resource Updating/Deleting/Failed"
```

Comment:
I think provisioning state should always be readonly. It is an outcome of a request sent to the server. For cancelling a deployment, the cancel request is an action. It's outcome is reflected in the provisioning state. Hence it would be safe to mark it as `"readonly": true` in all the swagger specs.
@markcowl @devigned @deepakrajendranmsft, @jianghaolu, @tbombach   - What do you think about this?